server/portal/apps/request_access/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
```

Comment:
Is this file used? I think this is not needed

Reply:
@jmartinez-tacc what do you think?

Reply:
Hi @rstijerina, you are right, I'll remove this file. Thank you!